charts/connect/templates/operator-deployment.yaml

```diff
@@ -68,17 +68,26 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "{{ .Values.operator.applicationName }}"
-            - name: OP_CONNECT_HOST
-              value: "{{- include "onepassword-connect.url" . }}"
             - name: POLLING_INTERVAL
               value: "{{ .Values.operator.pollingInterval }}"
+            - name: AUTO_RESTART
+              value: "{{ .Values.operator.autoRestart }}"
+            {{- if .Values.operator.serviceAccountToken.value }}
+            - name: OP_SERVICE_ACCOUNT_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.operator.serviceAccountToken.name }}
+                  key: {{ .Values.operator.serviceAccountToken.key }}
+            {{- end }}
+            {{- if .Values.operator.token.value }}
+            - name: OP_CONNECT_HOST
+              value: "{{- include "onepassword-connect.url" . }}"
             - name: OP_CONNECT_TOKEN
               valueFrom:
                 secretKeyRef:
                   name: {{ .Values.operator.token.name }}
                   key: {{ .Values.operator.token.key }}
-            - name: AUTO_RESTART
-              value: "{{ .Values.operator.autoRestart }}"
+            {{- end }}
```

Comment:
Is this right? Should it not be `if .Values.operator.token.key`, as there is no `token.value` field used anywhere as far as I can tell?